dev/RELEASE_PROCESS.rst

```diff
@@ -81,6 +105,19 @@ This document describes the process for creating new releases.
    finds the correct Ray version, and successfully runs some simple scripts on
    both MacOS and Linux as well as Python 2 and Python 3.
 
+10. **Create a GitHub release:** Create a GitHub release through the `GitHub
+   website`_. The release should be created at the commit from the previous
+   step. This should include **release notes**. Copy the style and formatting
+   used by previous releases. Create a draft of the release notes containing
+   information about substantial changes/updates/bugfixes and their PR number.
```

Comment:
Add a note about following the style of the previous release notes.

Reply:
It does say this 2 lines above. Should I move it here?